instantly/metadata/scraping.go

```diff
@@ -0,0 +1,27 @@
+package metadata
+
+import (
+	_ "embed"
+	"path/filepath"
+	"runtime"
+
+	"github.com/amp-labs/connectors/tools/scrapper"
+)
+
+var (
+	// Static file containing a list of object metadata is embedded and can be served.
+	//
+	//go:embed schemas.json
+	schemas []byte
+
+	FileManager = scrapper.NewMetadataFileManager(schemas, locator{}) // nolint:gochecknoglobals
+)
+
+type locator struct{}
+
+func (locator) AbsPathTo(filename string) string {
```

Comment:
repeated code. Please move to common package. and reuse. Also apply to other connectors in follow up PR.

Reply:
Please, review: https://github.com/amp-labs/connectors/pull/973